libraries/core_libs/consensus/src/dag/dag.cpp

```diff
@@ -475,12 +475,16 @@ void DagManager::getGhostPath(std::vector<blk_hash_t> &ghost) const {
 }
 
 // return {period, block order}, for pbft-pivot-blk proposing
-std::pair<uint64_t, std::vector<blk_hash_t>> DagManager::getDagBlockOrder(blk_hash_t const &anchor) {
+std::pair<uint64_t, std::vector<blk_hash_t>> DagManager::getDagBlockOrder(blk_hash_t const &anchor, uint64_t period) {
```

Comment:
I know it is not part of your changes, but do we need to return period? I have checked the code and it seems to me, that is used just for test and never for actual node. Maybe we could remove it and make it simpler

Reply:
I mean period in return std::pair to be clear

Reply:
Done